care-o-bot/105dc16/105dc16.bug

```diff
@@ -45,4 +57,4 @@ fix:
     - cob_control/cob_twist_controller/CMakeLists.txt
   languages:
     - CMake
-  time: 2014-11-21 (12:57)
+  time: 2014-11-21T12:57:00Z
```

Comment:
Please note again: unless the original `time` was already in UTC, this is not a valid conversion.

It's likely these times were reported in CET or CEST, which would make this `+200` (but this is just a guess).

Reply:
I have not worked on that. As I mentioned in the comment, I have worked only on fault and failure codes.

Reply:
I'm not sure I understand your comment: are you saying you didn't change these `time` entries?

What makes them show up in the diff here?

Reply:
Does this PR build on #359? See also https://github.com/robust-rosin/robust/pull/359#issuecomment-709039469, where I commented on 0c467070.

Reply:
Yes this PR is behind https://github.com/robust-rosin/robust/pull/359. I have modified the time yet after your comment.

Reply:
To prevent we accidentally merge this, perhaps drop 0c46707 for now?

Reply:
Just to avoid the confusion and separate the concerns, I will remove https://github.com/robust-rosin/robust/commit/0c4670703dfd75ccd4e25e0333b639c3e229917d, so https://github.com/robust-rosin/robust/pull/359 is only about the script and the schema.